waspc/examples/todoApp/ext/queries.js

```diff
@@ -19,8 +19,15 @@ export const getTask = async ({ id }, context) => {
   }
 
   const Task = context.entities.Task
-  const task = await Task.findOne(
-    { where: { id, user: { id: context.user.id } } }
-  )
+  // NOTE(matija): we can't call findOne() with the specific user, so we have to fetch user first
+  // and then manually check.
+  const task = await Task.findOne({ where: { id }, include: { user: true } })
+  if (!task) {
+    throw new HttpError(404)
+  }
+  if (task.user.id !== context.user.id) {
+    throw new HttpError(403)
```

Comment:
Aha ok, well this is maybe better approach because it returns 403. On the other hand, we just let them know that such task exists, which is maybe not so god.